src/main/resources/net/uaznia/lukanus/hudson/plugins/gitparameter/GitParameterDefinition/config.properties

```diff
@@ -13,4 +13,4 @@ parameter.default.value=Default Value
 parameter.selected.value=Selected Value
 parameter.use.repository=Use repository
 parameter.quick.filter=Quick Filter
-parameter.list.size=List Size
\ No newline at end of file
+parameter.list.size=List Size
```

Comment:
I don't see any new line which we must remove.
https://github.com/jenkinsci/git-parameter-plugin/blame/master/src/main/resources/net/uaznia/lukanus/hudson/plugins/gitparameter/GitParameterDefinition/config.properties
In this file and in other.

Reply:
I added a newline to the end of the file, I've been requested in the past to do so